timalsinab/Final-SEO-project
# Chatbot Integration

Why:
As a user, I would like to have a tutor to help my learning experience.

How:
Add bot.py file to initialize and set up the openai API. Created a chatbot.html file to display the messaging system and ping the API when I user sends a message.

Test Plan:
1. create an .env file in your project directory and add the API key that was shared to you.
2. run the app.py and navigate to AI tutor in the navbar
3. test texting messages to the chatbot and there should be active responses
